www/advanced-data-model.html

```diff
@@ -44,14 +49,9 @@ <h3>Advanced Data Models with OpenSeadragon</h3>
     that must be called from <code>downloadTileStart</code> in all cases
     except when aborting the job) notifies the ImageLoader about the job
     completion and status, and the data (if any) gets pulled into the system.
-</p>
-<p>
-    Last but not least, other functions may be necessary
-    to override, especially if you change the default tile data type
-    (<code>Image</code>). <b>Changing the data type means you have to override
-    all methods that perform tile caching</b> (<code>*TileCache*()</code>
-    like signature). Overriding cache key generation may or might not be
-    necessary.
+    Upon data submission, you must provide a data type you submit, and
+    optionally provide type convertors in case your format is not supported
```

Comment:
I've just noticed this (despite having seen the code before many times): the standard English spelling is "converter". Evidently "convertor" is also legal, but much less common. Perhaps we should rename the feature to align with common English spelling? Apologies for not noticing this sooner!

Reply:
Also in the whole codebase? I think variables/classes have the name too.

Reply:
Yeah, I think so. Again, apologies for not catching it sooner!